pom.xml

```diff
@@ -271,11 +273,9 @@
   <profiles>
     <profile>
       <id>release</id>
-      <properties>
-        <jdkVersion>1.5</jdkVersion>
-      </properties>
```

Comment:
Is this a related change? What are the implications?

Reply:
This is not related, it's a drive-by. I borrowed parts of `pom.xml` from another branch I'm working on and it included this change. I'm happy to leave this here though- I think it is strange to build our release in 1.5 when we dev in 1.6.